strutil/strutil.go

```diff
@@ -47,6 +48,18 @@ func MakeRandomString(length int) string {
 	return out
 }
 
+// Convert the given size in btes to a readable string
+func SizeToStr(size int64) string {
+	suffixes := []string{"B", "kB", "MB", "GB", "TB", "PB", "EB"}
```

Comment:
Why `kB` but `MB`? `2**10` vs `10 ** 3` aside, shouldn't those be uniformly capitalised?